appendices.tex

```diff
@@ -202,41 +203,105 @@ \subsubsection{Migrating your ledger}
 \section{Migrating between PostgreSQL versions}
 \label{sec-migration-postrgesql}
 
-If you're migrating between PostgreSQL versions, there are a few things to take
-into account.
+There are generally 2 ways to upgrade PostgreSQL\index{PostgreSQL upgrade} that apply to normal LSMB installations. 
+Both are described in the current PostgreSQL documentation at \url{https://www.postgresql.org/docs/current/index.html}.
+\begin{enumerate}
+    \item Dump\index{dump} and restore\index{restore} -
+        Described in more detail below.
+        Note that the roles must be restored before database creation and the data restore.
+    \item Upgrade in place using \texttt{pg\_upgrade}\index{pg\_upgrade} - When doing an upgrade in place, there are no specific LedgerSMB requirements.
+        Beware, upgrade in place requires both versions of the PostgreSQL to be installed and working. It is therefore more complex.
+\end{enumerate}
+
+LedgerSMB versions prior to version 1.9.16 did not support PostgreSQL 14 or higher due to a backward incompatible change in PostgreSQL 14.

```

Comment:
Maybe we should advise that anybody upgrading to Pg14 needs to upgrade to at least LSMB 1.10 first?